book/quick-brick/Setup.md

```diff
@@ -5,57 +5,151 @@
 ### React - node.js
 
 QuickBrick runs on node.js ^12.0.x 
-Please refer to [the environment setup guide](/dev-env/node.md) to set up your node environment
+Please refer to the [environment setup guide](/dev-env/node.md) to setup your node environment.
+
+To prepare a project for quick-brick you need to [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the official [QuickBrick](https://github.com/applicaster/QuickBrick) repository. This Repo contains the source code for the Quick Brick Project, which enables to create a Zapp App with a 100% react-native-flavored UI.
```

Comment:
It is never needed to fork the qb repo

Reply:
I forgot to remove this file. 
The new docs are native-project-dev.md and plugin-setup.md.
Sorry about that.

Reply:
Why `native` project dev ? 
I think this is generating more confusion than anything. There should be only way to run a QB-based app, which allows to work on either native or react-native code. I'm not sure that opposing react-native vs native is helpful